### ISSUE TITLE
Several syntax bugs in confluence.py update_page method

### ISSUE THREAD

**Issue by UTAS-Nick-Howden** [MattAgile/atlassian-python-api#6]
There are several syntax errors in the latest update_page method in confluence.py.
The code below resolves these issues this does make parent_id none optional again but I didn't want to change the method signature

```python
    def update_page(self, parent_id, page_id, title, body, type='page'):
        log.info('Updating {type} "{title}"'.format(title=title, type=type))
        if self.is_page_content_is_already_updated(page_id, body):
            return self.get_page_by_id(page_id)
        else:
            version = self.history(page_id)['lastUpdated']['number'] + 1

            data = {
                'id': page_id,
                'type': type,
                'title': title,
                'body': {'storage': {
                    'value': body,
                    'representation': 'storage'}},
                'version': {'number': version}
            }

            if parent_id is not None:
                data['ancestors'] = [{'type': 'page', 'id': parent_id}]
            return self.put('/rest/api/content/{0}'.format(page_id), data=data)
```

**Comment by MattAgile** [comment]
Thanks.
That was a merge of someone Pull Request.
I will check this out!

**Comment by MattAgile** [comment]
fixed in 65cc58205b0e71c004548334d3f4ed82da0aab61
Thanks for your feedback.
